api/src/Doctrine/NameSpaceIgnoringVersionComparator.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Doctrine;
+
+use Doctrine\Migrations\Version\Comparator;
+use Doctrine\Migrations\Version\Version;
+
+class NameSpaceIgnoringVersionComparator implements Comparator {
+    public function compare(Version $a, Version $b): int {
+        return strcmp(self::versionWithoutNamespace($a), self::versionWithoutNamespace($b));
+    }
+
+    private static function versionWithoutNamespace(Version $version): string {
+        $path = explode('\\', (string) $version);
```

Comment:
Falls wir je Migrations von einem Symfony Bundle bekommen könnte das zu Konflikten führen oder? Aber wenn ich [dieses Issue](https://github.com/doctrine/DoctrineMigrationsBundle/issues/152) richtig verstehe ist es bei Symfony eher üblich, alle Migrations in der Applikation zu haben, und allfällige Bundle-Migrations würden wir dann nicht aus dem Bundle laden sondern vom Bundle in unser migrations-Directory kopieren.